Python/DZ/DZ5/dz1.py

```diff
@@ -0,0 +1,16 @@
+# Задача 26:
+# Напишите программу,которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+#
+# *Пример:*
+#
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8
+
+def my_pow(a, b):
+    if b == 1:
+        return a
+    b -= 1
+    return a * my_pow(a, b)
```

Comment:
Почему бы это не совместить?
```suggestion
    return a * my_pow(a, b - 1)
```